newrelic.yml

```diff
@@ -23,628 +23,630 @@ common: &default_settings
   # debug.
   log_level: info
 
-# All of the following configuration options are optional. Review them, and
-# uncomment or edit them if they appear relevant to your application needs.
+  # All of the following configuration options are optional. Review them, and
```

Comment:
I think this would be much shorter if grpc was merged with dev

Reply:
Good call. Dev merged in with [d7756ab](https://github.com/newrelic/newrelic-ruby-agent/pull/1283/commits/d7756aba505709c581070f7ed16dc5eb54c3684d)